vdp/pipeline/v1alpha/pipeline_public_service.proto

```diff
@@ -154,4 +176,17 @@ service PipelinePublicService {
     };
     option (google.api.method_signature) = "name";
   }
+
+  // *Longrunning operation methods
+
+  // GetPipelineOperation method receives a
+  // GetPipelineOperationRequest message and returns a
+  // GetPipelineOperationResponse message.
+  rpc GetPipelineOperation(GetPipelineOperationRequest)
+      returns (GetPipelineOperationResponse) {
+    option (google.api.http) = {
+      get : "/v1alpha/{name=pipelines/*}/trigger-async/{name=operations/*}"
```

Comment:
i am not sure with two `name={...}` in the endpoint will cause any trouble.

Reply:
Yeah, I'm confused about it too. Should we just use `/v1alpha/{name=operations/*}`?

Reply:
> But if so, maybe we need a standalone operation backend to serve all the long running operations from different backend.

Any endpoint design that we can avoid this? We have too many backends now.

After model-backend refactoring, I think the `/v1alpha/{name=operations/*}` endpoint in `model-backend` will be removed eventually from VDP?

How about we refactor to the following for now? (not the best choice maybe)
```/v1alpha/{name=triggerAsyncOperations/*}```